package/plugin-gargoyle-usb-storage-noshare/Makefile

```diff
@@ -35,6 +35,7 @@ define Package/plugin-gargoyle-usb-storage-noshare
 	DEPENDS+=+kmod-nls-base +kmod-nls-cp1250 +kmod-nls-cp1251 +kmod-nls-cp437 +kmod-nls-cp775 +kmod-nls-cp850 
 	DEPENDS+=+kmod-nls-cp852 +kmod-nls-cp866 +kmod-nls-iso8859-1 +kmod-nls-iso8859-13 +kmod-nls-iso8859-15 
 	DEPENDS+=+kmod-nls-iso8859-2 +kmod-nls-koi8r +kmod-nls-utf8 +disktype +kmod-fuse +ntfs-3g
+	DEPENDS+=+kmod-usb-net-ipheth +usbmuxd
```

Comment:
We shouldn't tie this to usb-storage-noshare

Reply:
Yeah, will try to create a new plugin. My thought was that the other modems are already located here, but I see the point when usbmuxd actually pulls that much (didn't measure yet).

Reply:
My suggested patch has this already done for you :)